crowbar_framework/app/models/service_object.rb

```diff
@@ -919,6 +919,35 @@ def self.proposal_to_role(proposal, bc_name)
     RoleObject.new role
   end
 
+  #  we can speed-up the application of (n+1)th role if both(n,n+1)
+  #  roles are being applied on the same node.
+  #
+  #  eg. In our 2node deployment ceilometer{server,central} are always
+  #  applied on the same node, given that they have different priorities,
+  #  they are to be applied, one after the other.
```

Comment:
It would be helpful to explicitly mention `element_run_list_order` here and explain its effect on the run-list, otherwise the reader will be wondering why/how they end up with different priorities.